packages/mock-account-provider/src/parse_config.ts

```diff
@@ -0,0 +1,34 @@
+import { parse } from 'yaml'
```

Comment:
This can be used by the Remix app to load the config. Doesn't pull in node-fetch or anything unnecessary.